python/k4FWCore/ApplicationMgr.py

```diff
@@ -110,10 +111,15 @@ def _setup_reader(self, reader, iosvc_props):
     def fix_properties(self):
         # If there isn't an EventLoopMgr then it's the default
         # This will suppress two warnings about not using external input
-        try:
-            self._mgr.EventLoop
-        except AttributeError:
-            self._mgr.EventLoop = EventLoopMgr(Warnings=False)
+        if not hasattr(self._mgr, "EventLoop"):
+            self._mgr.EventLoop = EventLoopMgr()
+            try:
+                # Only available in Gaudi >= 40.0.0, when the Warnings attribute is removed
+                from Gaudi import __version__  # noqa: F401
```

Comment:
Does this mean that `Gaudi.__version__` is only available from Gaudi 40.0.0 onwards? Does `self._mgr.EventLoop.OutputLevel` not raise an `AttributeError` that we could catch instead of relying on this import?